lib/resources/majors.tsx

```diff
@@ -5164,6 +5164,201 @@ const bsNeuro: Major = {
   ],
 };
 
+//https://e-catalogue.jhu.edu/engineering/full-time-residential-programs/degree-programs/leadership-education/entrepreneurship-management-minor/#requirementstext
+const minorEntrMgmt: Minor = {
+  degree_name: 'Minor Entrepreneurship and Management',
+  abbrev: 'Minor Entr & Mgmt',
+  department: 'EN Center for Leadership Education',
+  total_degree_credit: 21,
+  url: 'https://e-catalogue.jhu.edu/engineering/full-time-residential-programs/degree-programs/leadership-education/entrepreneurship-management-minor/#requirementstext',
+  wi_credit: 0,
+  distributions: [
+    {
+      name: 'Statistics',
+      required_credits: 3,
+      min_credits_per_course: 3,
+      description:
+        'These courses expose students to the foundations of statistics that are used extensively in business ' +
+        'decision-making. These topics include correlation, estimation, hypothesis testing, linear regression, prediction, ' +
+        'and forecasting. Students may take either two elementary statistics courses or one intermediate/advanced calculus-based' +
+        ' statistics course from the list below.',
+      criteria:
+        'AS.230.205[C]^OR^AS.280.345[C]^OR^AS.280.345[C]^OR^EN.553.111[C]^OR^EN.553.112[C]^OR^EN.661.380[C]^OR^' +
+        'EN.553.211[C]^OR^EN.553.310[C]^OR^EN.553.311[C]^OR^EN.553.430[C]^OR^EN.560.240[C]',
+      fine_requirements: [
+        {
+          required_credits: 8,
+          description:
+            '<b>Option One: Two Elementary Statistics Courses.</b> Credit earned for AP Statistics (equivalent to EN.553.111)' +
+            ' will satisfy the first of the two required courses.<br />' +
+            'First course (select one): <br/>' +
+            'AS.230.205 Introduction to Social Statistics' +
+            '<br /> <i>OR</i> <br />' +
+            'AS.280.345 Public Health Biostatistics' +
+            '<br /> <i>OR</i> <br />' +
+            'EN.553.111 Statistical Analysis I <br/>' +
+            'Second course (select one) <br/>' +
+            'EN.553.112 Statistical Analysis II' +
+            '<br /> <i>OR</i> <br />' +
+            'EN.661.380 Decision Analytics <br/>',
+          criteria:
+            'AS.230.205[C]^OR^AS.280.345[C]^OR^AS.280.345[C]^OR^EN.553.111[C]^OR^EN.553.112[C]^OR^EN.661.380[C]',
```

Comment:
herro miseok :3 is AS.280.345 supposed to be here twice?

Reply:
prolly not, but it doesn't affect the logic kewk